### ISSUE TITLE
Trouble with browse/read all mode in Firefox and NVDA

### ISSUE THREAD

**Issue by himedlooff** [humaan/Modaal#81]
Steps to reproduce:
- I'm testing the demo page http://www.humaan.com/modaal/ on Firefox V50.1.0 with NVDA 2016.4
- When the page loads I am able to use the up/down cursor keys to browse the text (browse mode)
- When I activate the modal I am no longer able to use the cursors to browse the text
- "Read all starting at current position" doesn't work either (NVDA + Numpad +)
- Tabbing is fine

I expect to be able to read the contents of the modal.

Note that this might be an issue with my lack of experience using screen readers.

**Comment by danhumaan** [comment]
Hi @himedlooff thanks for raising the issue, we'll look into this one and see what we can reproduce.

the Modaal project has been a little quiet lately with other priorities take focus, but we'll update this as soon we get the chance to review. Thanks.

**Comment by danhumaan** [comment]
HI @himedlooff, thanks for your patience in us getting to this task. I believe this is now resolved following the implemented fix from #87. Please keep an eye out for updates in our next release.
